### PR TITLE
Display form errors in toast

### LINKED ISSUE #50: Why there are no toasts when error appears?
Why there are no toasts when error appears?

_Originally posted by @ref-humbold in https://github.com/ksiuwr/zosia16-site/pull/48_

### PR DESCRIPTION
**Pull request by ref-humbold**
Displays form field errors in toast. Fix #50

### CODE REVIEW

File: app/src/utils/forms.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from django.utils.safestring import mark_safe
+
+
+def errors_format(form):
+    infos = ["<span><strong>" + field + "</strong><br/>" + "<br/>".join(errs) + "</span>"
+             for field, errs in form.errors.items()]
+
+    return mark_safe("<p>" + "<br/><br/>".join(infos) + "</p>")
```
Comment (Bozydarek) [inline app/src/utils/forms.py]: Is it possible that `infos` will contains **any** output that originally will comes from users? If that's true you need to escape characters in that string first.

File: app/src/blog/views.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
-from django.views.decorators.http import require_http_methods
 from django.contrib.admin.views.decorators import staff_member_required
-from django.shortcuts import render, redirect
-from .models import BlogPost
+from django.shortcuts import redirect, render
+from django.views.decorators.http import require_http_methods
+
 from .forms import BlogPostForm
+from .models import BlogPost
```
Comment (Bozydarek) [inline app/src/blog/views.py]: I'm not an Python expert or even a Python developer, but shouldn't we prefer absolute imports over relative ones? Again I'm not the expert and I don't know which one is "cleaner".

Comment (ref-humbold) [reply app/src/blog/views.py]: PEP8 states that ["Absolute imports are recommended, (...) however, explicit relative imports are an acceptable alternative to absolute imports (...)"](https://www.python.org/dev/peps/pep-0008/#imports). I tend to convert relative imports to absolute ones, but PyCharm possibly tries to match new imports to existing ones.

File: app/src/utils/forms.py (hunk 1 above)
Comment (ref-humbold) [reply app/src/utils/forms.py]: It depends on what you put into errors. I think it should contain no user input, but additional escaping won't hurt ;)

File: app/src/blog/views.py (hunk 2 above)
Comment (ref-humbold) [reply app/src/blog/views.py]: I think I can change (almost) all relative imports to absolute, but this change could be applied directly on master.

**Pull request merged**